src/deepsparse/yolo/README.md

```diff
@@ -0,0 +1,155 @@
+wget -O abbey_road.jpg  https://upload.wikimedia.org/wikipedia/en/4/42/Beatles_-_Abbey_Road.jpg
+```
+
+[List of the YOLOv5 SparseZoo Models](
+https://sparsezoo.neuralmagic.com/?domain=cv&sub_domain=detection&page=1)
+
+```python
+from deepsparse.pipeline import Pipeline
+
+model_stub = "zoo:cv/detection/yolov5-l/pytorch/ultralytics/coco/pruned-aggressive_98"
+images = ["abbey_road.jpg"]
+
+od_pipeline = Pipeline.create(
+    task="yolo",
+    model_path=model_stub,
+)
+
+pipeline_outputs = od_pipeline(images=images, iou_thres=0.6, conf_thres=0.001)
+```
+
+### Annotate CLI
+You can also annotate an image source directly with the following CLI command:
+```bash
+deepsparse.object_detection.annotate --source abbey_road.jpg #Try --source 0 to annotate your live webcam feed
+```
+
+### DeepSparse Server
+As an alternative to Python API, the DeepSparse Server allows you to serve ONNX models and pipelines in HTTP.
+Configs for the server support the same arguments as the above pipelines.
+
+An example of starting and requesting a DeepSparse Server for YOLOv5 is given below.
+
+#### Installation
+The deepsparse server requirements can be installed by specifying the `server` extra dependency when installing
+DeepSparse.
+
+```bash
+pip install deepsparse[server]
+```
+
+#### Spinning Up
+The DeepSparse server supports the same tasks and model paths as the pipeline examples above.  The following
+uses the `deepsparse.server` script to launch a YOLOv5s model server.
+
+```bash
+deepsparse.server \
+    --task yolo \
+    --model_path zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/pruned_quant-aggressive_94
+```
+
+Once the server is running, it will print a local address to the terminal which you can use
+in your browser to access the server GUI. You can use the interface to upload images
+for inference. 
+
+Alternatively, you can use the following bash command to upload the images, with an additional `-F 'request=@...` line for each image
+
+```bash
+curl -X 'POST' \
+  'http://localhost:5543/predict/from_files' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: multipart/form-data' \
+  -F 'request=@Beatles_-_Abbey_Road.jpg;type=image/jpeg'
+```
+
+### Benchmarking
+The mission of Neural Magic is to enable GPU-class inference performance on commodity CPUs. Want to find out how fast our sparse YOLOv5 ONNX models perform inference? 
+You can quickly do benchmarking tests on your own with a single CLI command!
+
+You only need to provide the model path of a SparseZoo ONNX model or your own local ONNX model to get started:
+
+```bash
+deepsparse.benchmark zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/pruned_quant-aggressive_94
+
+>> Original Model Path: zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/pruned_quant-aggressive_94
+>> Batch Size: 1
+>> Scenario: async
+>> Throughput (items/sec): 120.5551
+>> Latency Mean (ms/batch): 99.3043
+>> Latency Median (ms/batch): 99.1637
+>> Latency Std (ms/batch): 2.7053
+>> Iterations: 1210
+```
```

Comment:
might want to push latency here since we're not going into detail for the benchmark i.e. add `--scenario sync`

Reply:
Updated as suggested. Let me know if the output looks reasonable now @mgoin